.size-snapshot.json

```diff
@@ -0,0 +1,1356 @@
+    "minified": 205,
+    "gzipped": 129,
+    "treeshaked": {
+      "rollup": {
+        "code": 0,
+        "import_statements": 0
+      },
+      "webpack": {
+        "code": 951
+      }
+    }
+  },
+  "utils/throwOnUndefinedProperty/index.js": {
+    "bundled": 405,
+    "minified": 227,
+    "gzipped": 168,
+    "treeshaked": {
+      "rollup": {
+        "code": 0,
+        "import_statements": 0
+      },
+      "webpack": {
+        "code": 951
+      }
+    }
+  },
+  "shared-components/checkbox/index.js": {
+    "bundled": 1569,
+    "minified": 1091,
+    "gzipped": 515,
+    "treeshaked": {
+      "rollup": {
+        "code": 283,
+        "import_statements": 144
+      },
+      "webpack": {
+        "code": 1988
+      }
+    }
+  },
+  "utils/styles/buttonReset.js": {
+    "bundled": 93,
+    "minified": 86,
+    "gzipped": 93,
+    "treeshaked": {
+      "rollup": {
+        "code": 0,
+        "import_statements": 0
+      },
+      "webpack": {
+        "code": 951
+      }
+    }
+  },
+  "svgs/icons/checkmark-icon.svg.js": {
+    "bundled": 947,
+    "minified": 716,
+    "gzipped": 444,
+    "treeshaked": {
+      "rollup": {
+        "code": 67,
+        "import_statements": 67
+      },
+      "webpack": {
+        "code": 1662
+      }
+    }
+  },
+  "shared-components/indicator/style.js": {
+    "bundled": 2093,
+    "minified": 2043,
+    "gzipped": 1236,
+    "treeshaked": {
+      "rollup": {
+        "code": 615,
+        "import_statements": 215
+      },
+      "webpack": {
+        "code": 1766
+      }
+    }
+  },
+  "svgs/icons/arrow.svg.js": {
+    "bundled": 894,
+    "minified": 671,
+    "gzipped": 432,
+    "treeshaked": {
+      "rollup": {
+        "code": 67,
+        "import_statements": 67
+      },
+      "webpack": {
+        "code": 1649
+      }
+    }
+  },
+  "constants/typography/index.js": {
+    "bundled": 533,
+    "minified": 410,
+    "gzipped": 243,
+    "treeshaked": {
+      "rollup": {
+        "code": 225,
+        "import_statements": 57
+      },
+      "webpack": {
+        "code": 1223
+      }
+    }
+  },
+  "svgs/icons/chevron-icon.svg.js": {
+    "bundled": 934,
+    "minified": 705,
+    "gzipped": 428,
+    "treeshaked": {
+      "rollup": {
+        "code": 67,
+        "import_statements": 67
+      },
+      "webpack": {
+        "code": 1659
+      }
+    }
+  },
+  "shared-components/dialogModal/index.js": {
+    "bundled": 3611,
+    "minified": 2220,
+    "gzipped": 984,
+    "treeshaked": {
+      "rollup": {
+        "code": 1847,
+        "import_statements": 513
+      },
+      "webpack": {
+        "code": 3179
+      }
+    }
+  },
+  "shared-components/avatar/style.js": {
+    "bundled": 1729,
+    "minified": 1573,
+    "gzipped": 1065,
+    "treeshaked": {
+      "rollup": {
+        "code": 433,
+        "import_statements": 105
+      },
+      "webpack": {
+        "code": 1501
+      }
+    }
+  },
+  "shared-components/dropdown/index.js": {
```

Comment:
This is a useful file that I think we should be committing. Since it's not currently in the repo, it's all new, but I used it to try and track down why there was a very tiny change in bundle size, and it's due to `prettier` auto-formatting in `src/dropdown`. 

I will look into a way to more seamlessly integrate this into our git workflow. It only matters for releases so we don't need to enforce it at the individual PR level.